src/components/IterableEditComponent.tsx

```diff
@@ -547,7 +551,7 @@ export class IterableEditComponent extends React.Component<ImageUploadProps | It
                                 onMouseEnter={this.showIterableButtons.bind(this, index)}
                                 onMouseLeave={this.hideIterableButtons.bind(this, index)}>
                                 <input
-                                    disabled={getReadOnly(this.props.field.iterabletype.readonly, datum) || this.props.readonly}
+                                    disabled={this.checkReadonly(this.props.field.iterabletype.readonly, datum)}
```

Comment:
I think datum is same as currentModel, use currentModel only.

Reply:
Updated